lgc/builder/SubgroupBuilder.cpp

```diff
@@ -325,6 +325,38 @@ Value *SubgroupBuilder::CreateSubgroupShuffle(Value *const value, Value *const i
     // The ds_bpermute intrinsic requires the index be multiplied by 4.
     return CreateMapToInt32(mapFunc, value, CreateMul(index, getInt32(4)));
   }
+
+  if (supportPermLane64Dpp()) {
+    assert(getShaderWaveSize() == 64);
+
+    // Start the WWM section by setting the inactive lanes.
+    Value *const identity = Constant::getNullValue(value->getType());
+    Value *wwmValue = BuilderBase::get(*this).CreateSetInactive(value, identity);
```

Comment:
What happens if you use PoisonValue instead of a null value? Reading an inactive lane is undefined, so semantically it ought to be okay.